New Flake 8 errors

https://ci.ros2.org/job/ci_linux-aarch64/1426/testReport/

```
./ros2cli/plugin_system.py:63:5: F841 local variable 'e' is assigned to but never used
    except PluginException as e:
    ^

./ros2cli/plugin_system.py:68:5: F841 local variable 'e' is assigned to but never used
    except Exception as e:
    ^

./ros2cli/entry_points.py:86:9: F841 local variable 'e' is assigned to but never used
        except Exception as e:
        ^

3     F841 local variable 'e' is assigned to but never used

22 files checked
3 errors
```

--- a/ros2cli/ros2cli/entry_points.py
+++ b/ros2cli/ros2cli/entry_points.py
@@ -83,7 +83,7 @@ def load_entry_points(group_name):
     for entry_point in get_entry_points(group_name).values():
         try:
             extension_type = entry_point.load()
-        except Exception as e:
+        except Exception as e:  # noqa: F841
             logger.warn(
                 "Failed to load entry point '{entry_point.name}': {e}"
                 .format_map(locals()))

--- a/ros2cli/ros2cli/plugin_system.py
+++ b/ros2cli/ros2cli/plugin_system.py
@@ -60,12 +60,12 @@ def _instantiate_extension(
 
     try:
         extension_instance = extension_class()
-    except PluginException as e:
+    except PluginException as e:  # noqa: F841
         logger.warn(
             "Failed to instantiate '{group_name}' extension "
             "'{extension_name}': {e}".format_map(locals()))
         return None
-    except Exception as e:
+    except Exception as e:  # noqa: F841
         logger.error(
             "Failed to instantiate '{group_name}' extension "
             "'{extension_name}': {e}".format_map(locals()))

--- a/ros2service/ros2service/verb/call.py
+++ b/ros2service/ros2service/verb/call.py
@@ -78,7 +78,7 @@ def requester(service_type, service_name, values, period):
 
     try:
         set_msg_fields(request, values_dictionary)
-    except SetFieldError as e:
+    except SetFieldError as e:  # noqa: F841
         return "Failed to populate field '{e.field_name}': {e.exception}" \
             .format_map(locals())
 

--- a/ros2topic/ros2topic/verb/pub.py
+++ b/ros2topic/ros2topic/verb/pub.py
@@ -90,7 +90,7 @@ def publisher(message_type, topic_name, values, node_name, period, once):
     msg = msg_module()
     try:
         set_msg_fields(msg, values_dictionary)
-    except SetFieldError as e:
+    except SetFieldError as e:  # noqa: F841
         return "Failed to populate field '{e.field_name}': {e.exception}" \
             .format_map(locals())
 

